resources/views/backend/event/create.blade.php

```diff
@@ -30,9 +33,9 @@
                     {!! Form::label('title', 'Title*', ['class' => 'col-md-2 col-form-label']) !!}
 
                     <div class="col-md-10">
-                        {!! Form::text('title', '', ['class' => 'form-control', 'required' => true]) !!}
```

Comment:
@IsharaEkanayaka is there any special reason to remove the required flag from this input?